static/app/views/organizationCreate/index.spec.tsx

```diff
@@ -9,6 +10,9 @@ describe('OrganizationCreate', function () {
   beforeEach(() => {
     ConfigStore.get('termsUrl');
     ConfigStore.get('privacyUrl');
+
+    // Set only a single region in the config store by default
+    ConfigStore.set('regions', [{name: '--monolith--', url: 'https://example.com'}]);
```

Comment:
You might be leaking state here. Should we backup state at the beginning of `beforeEach()` and restore in an `afterEach`?

Reply:
Good point, I'll go ahead and add that. Do we have global setup/teardown fixtures for ConfigStore by chance?

Reply:
>  Do we have global setup/teardown fixtures for ConfigStore by chance?

I don't think we have global setup/teardown on any of our reflux stores.